tests/mock_tests/dashhaorch_ut.cpp

```diff
@@ -0,0 +1,444 @@
+#include "mock_orch_test.h"
+#include "mock_table.h"
+#include "mock_sai_api.h"
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include "dash/dashhaorch.h"
+using namespace ::testing;
+
+EXTERN_MOCK_FNS
+
+namespace dashhaorch_ut 
+{
+    DEFINE_SAI_GENERIC_APIS_MOCK(dash_ha, ha_set, ha_scope);
+
+    using namespace mock_orch_test;
+
+    class DashHaOrchTest : public MockOrchTest
+    {
+    protected:
+
+        void ApplySaiMock()
+        {
+            INIT_SAI_API_MOCK(dash_ha);
+            MockSaiApis();
+        }
+
+        void PreTearDown() override
+        {
+            RestoreSaiApis();
+            DEINIT_SAI_API_MOCK(dash_ha);
+        }
+
+        void CreateHaSet()
+        {
+            auto consumer = unique_ptr<Consumer>(new Consumer(
```

Comment:
Just curious, why not use `swss::Table` and `m_dashHaOrch->addExistingData` like existing tests?

Reply:
No particular reason. Is `addExistingData` preferred?

Reply:
I noticed this `consumer` and `addToSync` are also commonly used in UTs, I think the benefit is we can explicitly set the `operation`?

Reply:
`Table` has `del` which should allow us to choose the op. But I didn't realize that we used `addToSync` so much, I think it's fine to leave this as-is then.